avro-fastserde/src/test/avro/defaultsTest.avsc

```diff
@@ -350,6 +350,54 @@
         }
       },
       "default": {"test": [null]}
+    },
+    {
```

Comment:
No code-gen results for these new fields in this PR?

Reply:
No, we currently don't add this to the codegen directory. Only some of the tests are put in there. Perhaps those should be added...